lib/iris/util.py

```diff
@@ -1794,3 +1794,37 @@ def mask_cube(cube, points_to_mask):
     cube.data = ma.masked_array(cube.data)
     cube.data[points_to_mask] = ma.masked
     return cube
+
```

Comment:
Check this:  Is identical to before, except for syntax over multiple lines  :)

Reply:
@abooton Yup, that's the magic of `black` :wink:

Reply:
:ghost: :imp: :see_no_evil: